tests-2015/Cargo.toml

```diff
@@ -11,6 +11,7 @@ edition = "2015"
 build = "../tests/src/build.rs"
 
 [lib]
+name = "tests"
```

Comment:
Why is this needed? It is not related to builders.

Reply:
This is needed so that the doctests I have added to the `tests` crate also work in these shim crates.